k8s/helm-charts/seldon-core-v2-setup/values.yaml

```diff
@@ -69,10 +69,11 @@ security:
           caPath: /tmp/certs/ddc/ca.crt
           serverCaPath: /tmp/certs/dds/ca.crt
 
-imagePullSecret:
+# A list of image pull secrets
+imagePullSecrets:
 
 opentelemetry:
-  endpoint: seldon-collector:4317
+  endpoint: seldon-collector.seldon-mesh:4317
```

Comment:
would this work still with per namespace install? I am assuming that users would need to provide another value instead?

Reply:
The assumption its in the manager install namespace but agree this needs to be changed to be based on install namespace

Reply:
will change so it uses namespace of install

Reply:
actually, the reason its hardwired is we don't know where you will install opentelemetry or kafka so its up to you to change it really.